internal/rbd/manager.go

```diff
@@ -268,3 +316,108 @@ func (mgr *rbdManager) CreateVolumeGroup(ctx context.Context, name string) (type
 
 	return vg, nil
 }
+
+func (mgr *rbdManager) CreateVolumeGroupSnapshot(
+	ctx context.Context,
+	vg types.VolumeGroup,
+	name string,
+) (types.VolumeGroupSnapshot, error) {
+	// TODO: reserve a unique name for the snapshot
+	pool, err := vg.GetPool(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	// volumeNamePrefix is an optional parameter, can be an empty string
+	prefix := mgr.parameters["volumeNamePrefix"]
```

Comment:
it should be snapshotNamePrefix or groupNamePrefix?

Reply:
`groupNamePrefix` seems most suitable, I'll use that.